simulator/console/console.go

```diff
@@ -0,0 +1,25 @@
+package console
+
+import (
+	"log"
+
+	socketio "github.com/googollee/go-socket.io"
+)
+
+type Console struct {
+	WebSocket socketio.Conn
+}
+
+func (c *Console) PrintConsole(message string) {
```

Comment:
PrintLog

Reply:
Find ich unlogisch dann würde ein richtiges Log-File erwartet werden

Reply:
log heisst nicht ein log file, es ist oft so, aber nicht immer, eig in modernen systemen ist ein log immer auf stdout und die umgebung entscheindet wo es dann gespeichert wird oder gezeigt usw....